frontend/common/src/helpers/storybookUtils.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Viewport,
+  ViewportStyles,
+  Styles,
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
This is so bad!  I haven't been able to figure out how to avoid this override.  Chromatic clearly doesn't care.